crates/agglayer-prover/src/executor.rs

```diff
@@ -227,22 +230,18 @@ impl Service<Request> for LocalExecutor {
     fn call(&mut self, req: Request) -> Self::Future {
         let prover = self.prover.clone();
         let stdin = req.into();
-        let opts = ProofOpts {
-            timeout: Some(self.timeout),
-            ..Default::default()
-        };
-        let kind = SP1ProofKind::Plonk;
 
         let proving_key = self.proving_key.clone();
         let verification_key = self.verification_key.clone();
 
         debug!("Proving with CPU prover with timeout: {:?}", self.timeout);
         Box::pin(
             spawn_blocking(move || {
-                let context = SP1Context::default();
                 debug!("Starting the proving of the requested MultiBatchHeader");
                 let proof = prover
-                    .prove(&proving_key, stdin, opts, context, kind)
+                    .prove(&proving_key, &stdin)
+                    .plonk()
+                    .run()
```

Comment:
```suggestion
                    .prove(&proving_key, &stdin)
                    .plonk()
                    .timeout(self.timeout)
                    .run()
```

Considering the code below that calls it this way, this is probably the right syntax to not lose the timeout that was previously defined in `opts`.